### PR TITLE
Kickoff

### PR DESCRIPTION
**Pull request by mcolombo87**
### Summary
First Pull Request with Ruby on Rails base project and PostgreSQL database config.
### Trello Card
[Related Trello card](https://trello.com/c/kOIxnd63/15-crear-aplicaci%C3%B3n-rails-wbooksapi)
### ScreenShots
image

### CODE REVIEW

File: log/development.log [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Started GET "/" for 127.0.0.1 at 2018-11-23 14:59:32 -0300
```
Comment (floresv) [inline log/development.log]: You should ignore this files. :thinking:

Comment (mcolombo87) [reply log/development.log]: Done! ad0a182

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+# available as an environment variable. For example:
+#
+#   DATABASE_URL="postgres://myuser:mypass@localhost/somedatabase"
+#
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
+production:
```
Comment (facucompe) [inline config/database.yml:84]: Let's leave production like this:
```
production:
  url: <%= ENV['DATABASE_URL'] %>
```
Later we will use that url

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Folders ignored
```
Comment (facucompe) [inline .gitignore:29]: Add the default `.gitignore` file

**Review by facucompe — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (mcolombo87) [reply .gitignore:29]: Done! aa5545e

File: config/database.yml (hunk 2 above)
Comment (mcolombo87) [reply config/database.yml:84]: Done! 7c4a680

**Review by floresv — DISMISSED** [review]
Looks good to me

**Review by facucompe — APPROVED** [review]

**Pull request merged**